sdks/unreal/Agones/Source/Agones/Private/AgonesComponent.cpp

```diff
@@ -88,8 +95,8 @@ void UAgonesComponent::ConnectSuccess(const FGameServerResponse GameServerRespon
 void UAgonesComponent::Ready(const FReadyDelegate SuccessDelegate, const FAgonesErrorDelegate ErrorDelegate)
 {
 	TSharedRef<IHttpRequest> Request = BuildAgonesRequest("ready");
-	Request->OnProcessRequestComplete().BindLambda(
-		[SuccessDelegate, ErrorDelegate](FHttpRequestPtr HttpRequest, FHttpResponsePtr HttpResponse, const bool bSucceeded) {
+	Request->OnProcessRequestComplete().BindWeakLambda(
+		this, [SuccessDelegate, ErrorDelegate](FHttpRequestPtr HttpRequest, FHttpResponsePtr HttpResponse, const bool bSucceeded) {

```

Comment:
nit - can we keep the style consistent:
```
Request->OnProcessRequestComplete().BindWeakLambda(
		this, [SuccessDelegate
```
vs 
```
Request->OnProcessRequestComplete().BindWeakLambda(this, 
```
Will see what my clang formatter says (if anything)

Reply:
Yeah our project's clang is what decided on this, but happy to conform to your preference :)

Reply:
more that both are used ... super small nit